db/migrate/20220609221942_remove_orphaned_owners.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveOrphanedOwners < ActiveRecord::Migration[7.0]
+  def change
+    Ownership.includes(:user).where(user: {id: nil}).destroy_all
+  end
```

Comment:
What happened to adding a foreign key?

Reply:
Ah, I see from [this conversation in Slack](https://shopify.slack.com/archives/GM1850REJ/p1655242779870469) that the intention is to split the migration into two individual migrations.